apps/src/lab2/levelEditors/levelData/EditToolboxBlocks.tsx

```diff
@@ -0,0 +1,167 @@
+import classNames from 'classnames';
+import {isEqual} from 'lodash';
+import React, {useCallback} from 'react';
+
+import {Button} from '@cdo/apps/componentLibrary/button';
+import Checkbox from '@cdo/apps/componentLibrary/checkbox/Checkbox';
+import {BodyTwoText} from '@cdo/apps/componentLibrary/typography';
+import {BlockTypes} from '@cdo/apps/music/blockly/blockTypes';
+import {
+  categoryTypeToLocalizedName,
+  dynamicCategories,
+} from '@cdo/apps/music/blockly/toolbox';
+import {defaultMaps} from '@cdo/apps/music/blockly/toolbox/definitions';
+import {
+  Category,
+  CategoryBlocksMap,
+  ToolboxType,
+} from '@cdo/apps/music/blockly/toolbox/types';
+import {BlockMode} from '@cdo/apps/music/constants';
+import {getTypedKeys, ValueOf} from '@cdo/apps/types/utils';
+
+import MultiCategorySelect from './MultiCategorySelect';
+
+import styles from './edit-music-level-data.module.scss';
+
+// Levelbuilder-facing English block names.
+const blockDescriptions: {[blockType in BlockTypes | string]?: string} = {
+  [BlockTypes.PLAY_SOUND_AT_CURRENT_LOCATION_SIMPLE2]: 'Play Sound',
+  [BlockTypes.PLAY_PATTERN_AT_CURRENT_LOCATION_SIMPLE2]: 'Play Pattern',
+  [BlockTypes.PLAY_CHORD_AT_CURRENT_LOCATION_SIMPLE2]: 'Play Chord',
+  [BlockTypes.PLAY_REST_AT_CURRENT_LOCATION_SIMPLE2]: 'Rest',
+  [BlockTypes.TRIGGERED_AT_SIMPLE2]: 'Trigger',
+  [BlockTypes.PLAY_SOUNDS_TOGETHER]: 'Play Together',
+  [BlockTypes.PLAY_SOUNDS_SEQUENTIAL]: 'Play Sequential',
+  [BlockTypes.PLAY_SOUNDS_RANDOM]: 'Play Random',
+  [BlockTypes.REPEAT_SIMPLE2]: 'Repeat',
+  [BlockTypes.SET_VOLUME_EFFECT_AT_CURRENT_LOCATION_SIMPLE2]: 'Set Volume',
+  [BlockTypes.SET_FILTER_EFFECT_AT_CURRENT_LOCATION_SIMPLE2]: 'Set Filter',
+  [BlockTypes.SET_DELAY_EFFECT_AT_CURRENT_LOCATION_SIMPLE2]: 'Set Delay',
+};
```

Comment:
Should we unify this list with the music code?

Reply:
It also looks like we are only defining block text here for blocks in Simple2. For advanced mode, it looks like we're falling back to the types. That's a good behavior in case we forget to add to this for a new block, but we might want to take care of the current Advanced Mode blocks now.

Reply:
Yeah good call, had added the Simple2 blocks to test this out but forgot to back and add advanced.
Re: unifying with music code, yeah I could move this into /music and/or combine with the existing definitions in toolboxBlocks.ts? Just noting that these are only levelbuilder-facing strings because otherwise the block name comes from our JSON definitions and has placeholder values

Reply:
> Re: unifying with music code, yeah I could move this into /music and/or combine with the existing definitions in toolboxBlocks.ts? Just noting that these are only levelbuilder-facing strings because otherwise the block name comes from our JSON definitions and has placeholder values

I think it's reasonable for the music app to contain an extra internal english-only string for each block that is only shown to levelbuilders.  `toolboxBlocks.ts` could be good, so that we have a single place to define each block.

Reply:
Moved to toolboxBlocks.ts